tgui/packages/tgui/styles/themes/abductor.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+ @use 'sass:color';
```

Comment:
Do you need this copyright license for this presumably custom scss?

Reply:
Discussed this via discord, we decided to leave this in just in case.